odk1-src/central-install-digital-ocean.rst

```diff
@@ -0,0 +1,235 @@
+
+   .. image:: /img/central-install/nano.png
+
+Now, we will bundle everything together into a server. Type ``docker-compose build`` and press **Enter** to do this. This will take a long time and generate quite a lot of text output. Don't worry if it seems to pause without saying anything for a while. When it finishes, you should see some "Successfully built" type text and get your input prompt back. When that happens, type ``docker-compose create`` and press **Enter**.
+
+Once that is complete, congratulations! You have installed your copy of ODK Central. Next, we need to teach the server how to start it up, and do so.
+
+.. _central-install-digital-ocean-startup:
+
+Starting up ODK Central
+-----------------------
+
+Now, we want to run your new ODK server software. But we don't want to only run it once: if we do that, then if your server machine crashes or restarts, the software won't start back up. We want to tell the machine to always run the server. To teach it to do this, we have to copy a file to the right spot. To do this, run ``cp files/docker-compose@.service /etc/systemd/system``.
+
+Once that's done, run ``systemctl start docker-compose@central`` (type it in and press **Enter**) to start Docker, which will then load the ODK server. The first time you start it, it will take a while to set itself up. Once you give it a few minutes and you have input control again, you'll want to see whether everything is running correctly:
+
+ - To see if Docker itself is working correctly, you can run ``systemctl status docker-compose@central``. If you see text that says ``Active: active (running)`` then everything is working great.
+ - To see if ODK has finished loading inside of Docker, run ``docker-compose ps``. Under the ``State`` column, you will want to see text that reads ``Up (healthy)``. If you see ``Up (health: starting)``, give it a few minutes. If you see some other text, something has gone wrong.
+ - If your domain name has started working, you can visit it in a web browser to check that you get the ODK Central management website.
+
+Once we're finally sure that everything is working, run ``systemctl enable docker-compose@central``. This will make sure the ODK server is always running, even if something goes wrong or the machine reboots.
+
+You're almost done! All you have to do is create an Administrator account so that you can log into Central.
+
+.. _central-install-digital-ocean-account:
+
+Logging into ODK Central
+------------------------
+
+If visiting your server domain name address in your browser does not load the ODK Central management website, you may have to wait a few minutes or hours (possibly even a day) for the domain name itself to get working. These instructions are explained in further depth on the page detailing the :doc:`central-command-line`.
+
+Once you do see it working, you'll want to set up your first administrator account. To do this:
```

Comment:
I think Administrator should be capitalized?

Reply:
eh. why not